.travis.yml

```diff
@@ -3,18 +3,12 @@ rust:
 - beta
 - stable
 - nightly
-matrix:
-  include:
-  - rust: nightly-2018-07-22
-    env: CLIPPY=YESPLEASE
-    script:
-    - rustup component add clippy-preview
-    - cargo clippy -- -D warnings
-  - rust: 1.27.2
-    env: RUSTFMT=YESPLEASE
-    script:
-    - rustup component add rustfmt-preview
-    - cargo fmt --all -- --write-mode=diff
+
+script:
+- cargo fmt --all -- --write-mode=diff
+- cargo build --all --all-features
+- cargo test --all --all-features
```

Comment:
Did you mean to drop clippy?

Reply:
Yeah, but only temporarily until I figured out what to do with the codebase (maybe a big"ish" refactoring).